{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 4", "trace": "@pytest.mark.flaky(reruns=2, reruns_delay=2)\n    def test_demo():\n>       assert 3==4\nE       assert 3 == 4\n\ntest_pytest-rerun.py:33: AssertionError"}, "start": 1641970768442, "stop": 1641970768442, "uuid": "83a71281-dc8b-49ea-97c1-399f51ae1262", "historyId": "f2e80b3129cfcfce8f5b5deca556f768", "testCaseId": "7b98568a93d8f06328af6e6db02da49c", "fullName": "test_pytest-rerun#test_demo", "labels": [{"name": "tag", "value": "flaky(reruns=2, reruns_delay=2)"}, {"name": "suite", "value": "test_pytest-rerun"}, {"name": "host", "value": "liyinchideMacBook-Pro.local"}, {"name": "thread", "value": "89996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest-rerun"}]}